src/circle.rs

```diff
@@ -0,0 +1,59 @@
+use macroquad::math::{vec2, Rect, Vec2};
```

Comment:
This code got moved to https://docs.rs/macroquad/0.3.10/macroquad/math/struct.Circle.html#
If macroquad's circle is sufficient - I would prefer using it

Reply:
Ah right, must've slipped through when I merged